### ISSUE TITLE
TypeError: expected string or bytes-like object

### ISSUE THREAD

**Issue by masteradit** [dwisiswant0/apkleaks#37]
I was trying out apkleaks on an app that I had created using Flutter. But when I ran the command `apkleaks -f app-arm64-v8a-release.apk`, I got this error:

```
     _    ____  _  ___               _
    / \  |  _ \| |/ / |    ___  __ _| | _____
   / _ \ | |_) | ' /| |   / _ \/ _` | |/ / __|
  / ___ \|  __/| . \| |__|  __/ (_| |   <\__ \
 /_/   \_\_|   |_|\_\_____\___|\__,_|_|\_\___/
 v2.3.0
 --
 Scanning APK file for URIs, endpoints & secrets
 (c) 2020-2021, dwisiswant0

Traceback (most recent call last):
  File "C:\Python38\Scripts\apkleaks-script.py", line 11, in <module>
    load_entry_point('apkleaks==2.3.0', 'console_scripts', 'apkleaks')()
  File "c:\python38\lib\site-packages\apkleaks\cli.py", line 32, in main
    init = APKLeaks(args)
  File "c:\python38\lib\site-packages\apkleaks\apkleaks.py", line 29, in __init__
    self.disarg = re.split(r"\s|=", args.args)
  File "c:\python38\lib\re.py", line 231, in split
    return _compile(pattern, flags).split(string, maxsplit)
TypeError: expected string or bytes-like object
```

I ran this command in the same directory where my apk file is present. I am unable to figure out why I am getting this error.

**Comment by farhankn** [comment]
Getting the same issue for me. Ran on Python3  Ubunty 20.04

**Comment by dwisiswant0** [comment]
Hi, @farhankn & @masteradit. It should be fixed in v2.3.1.
